workflow/template-build-and-run-hpxml-with-stochastic-occupancy.osw

```diff
@@ -83,17 +83,17 @@
     },
     {
       "arguments": {
-        "hpxml_path": "../workflow/built.xml",
+        "hpxml_path": "../../../workflow/built.xml",
         "schedules_type": "stochastic",
-        "output_csv_path": "../stochastic.csv",
-        "hpxml_output_path": "../workflow/built-stochastic-schedules.xml"
+        "output_csv_path": "../workflow/run/stochastic.csv",
+        "hpxml_output_path": "../../../workflow/built-stochastic-schedules.xml"
```

Comment:
This seems odd. Why do they not go up the same number of directories to get to the workflow dir?

Reply:
`output_csv_path` is relative to the HPXML file? https://github.com/NREL/OpenStudio-HPXML/blob/relative-path-options-2/BuildResidentialScheduleFile/measure.rb#L127

Reply:
Shouldn't it be changed to be consistent with the other arguments?

Reply:
Yes, but requires minor change to `BuildResidentialScheduleFile` measure.